plugins/flashrom/flashrom.quirk

```diff
@@ -43,3 +43,6 @@ Plugin=flashrom
 FlashromProgrammer=lspcon_i2c_spi
 Name=PS175
 Vendor=Parade
+
+[FLASHROM-LSPCON-I2C-SPI\HwidFamily=Google_Hatch]
+FlashromLspconAuxDeviceName=DPDDC-B
```

Comment:
Any chance at publishing this somewhere more firmware-y rather than having to hardcode in `.quirk` eventually?  It feels like something that could go in ACPI or DT.

Reply:
It's a little weird, because ACPI already specifies which I2C bus the LSPCON is connected to but we need to use the DisplayPort aux channel to access DPCD- the LSPCON appears in two places, one of which is a control/debug interface and the other being mostly transparent.

That aside, updating firmware would be non-trivial since there are devices in the wild and no current plan to issue any updates (which would require re-validating).

Reply:
But would it be possible to add another field to "new" systems that could pick this up automatically?

Reply:
It probably could be, but I'm not sure exactly how that might look. Hanging something off the graphics pipe doesn't really work because that gets probed via PCI, but it might be possible to encode something like this quirk in a spare attribute on the I2C device.